scripts/variantstore/wdl/GvsAssignIds.wdl

```diff
@@ -26,24 +26,24 @@ workflow GvsAssignIds {
       dataset_name = dataset_name,
       datatype = "sample_info",
       schema_json = sample_info_schema_json,
-      max_table_id=1,
+      max_table_id = 1,
       superpartitioned = "false",
       partitioned = "false",
       service_account_json_path = service_account_json_path,
-      preemptible_tries = preemptible_tries,
+      preemptible_tries = create_tables_preemptible_override,
```

Comment:
ToL: for these very short, small VM tasks that alter the database should we just use "real" VMs and not have the PE option?  Then we also don't have to worry about getting pre-empted and handling that.

Reply:
I'm happy to remove it 👍🏻